point_num: 4

point_0_lat: 39.955166
point_0_lon: 116.263271
point_0_x: 0.0
point_0_y: 0.0

point_1_lat: 39.955336
point_1_lon: 116.263293
point_1_x: 0.0
point_1_y: 20.0

point_1_lat: 39.955315
point_1_lon: 116.263518
point_1_x: 20.0
point_1_y: 20.0

point_1_lat: 39.955142
point_1_lon: 116.263495
point_1_x: 20.0
point_1_y: 0.0
